src/ProspaAspNetCoreApiNsb/Infrastructure/Behaviours/LoggingBehavior.cs

```
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Serilog;
+
+namespace ProspaAspNetCoreApiNsb.Infrastructure.Behaviours
+{
+    public class LoggingBehavior<TRequest, TResponse> : IPipelineBehavior<TRequest, TResponse>
+    {
+        private readonly ILogger _logger = Log.ForContext<LoggingBehavior<TRequest, TResponse>>();
+
+        public async Task<TResponse> Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate<TResponse> next)
+        {
+            var traceId = Guid.NewGuid();
+
+            try
+            {
+                _logger.Verbose("Handling {TRequest}. TraceId: {TraceId} {UtcNow}", typeof(TRequest).FullName, traceId, DateTime.UtcNow);
+
+                var response = await next();
```

if a handler throws an exception why don't we just let it bubble up and have it logged there?
This will lead to double logging exceptions.
We already have exception handling middleware.

---

I've removed meditor logging behavior altogether